spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
```

Comment:
![image](https://github.com/mailsg/rails-blog-app/assets/93039359/987427e8-1c26-4994-8ddb-3a21a05f50f7)

- [ ] Can you kindly remove the Live Demo section since you don't have any working live link for your app 🤔 Once you have a working live link then you can add this LIve Dem section containing the live link :+1:

Reply:
👍